### ISSUE TITLE
Generate a manifest

### ISSUE THREAD

**Issue by boris-petrov** [webark/ember-cli-styles#5]
@webark as you requested in #1, I open a new issue on this topic. However, I read the code a bit and I don't understand why the manifest isn't simply:

```js
getOwner(this).lookup(`style:${componentName}`).styleNamespace
```

Isn't this *exactly* what `podNames` used to do in `ember-component-css`? This code seems to do exactly what I need. Is there any problem using that?

**Comment by webark** [comment]
I think this is fine for now. If you could update the commit message to reference this issue, we can close it once the other PR is merged.
